spec/views/devise/sessions/new.html.slim_spec.rb

```diff
@@ -26,7 +26,7 @@
 
     expect(rendered).
       to have_link(
-        t('links.create_account'), href: sign_up_email_path
+        t('links.create_account'), href: sign_up_email_url(request_id: nil)
```

Comment:
I just tried changing to `_path` and it failed. `_url` is required if you want to check for params as well.